src/IpDetails.php

```diff
@@ -36,31 +38,45 @@ private function __construct(
         string $threatLevel,
         string $isp
     ) {
-        $this->ipAddress = $ipAddress;
+        $this->ipAddress     = $ipAddress;
```

Comment:
I would transform this into a Constructor property promotion

Reply:
Yes, now that we support php8 it would be a nice improvement 👍

Reply:
Solved in https://github.com/ScullWM/dbip-client/pull/2/files